src/plugins/bugzilla_format.conf

```diff
@@ -55,5 +55,5 @@ Additional info:: \
 
 Truncated backtrace:: %bare_%short_backtrace
 
-%attach:: -comment,-reason,-reported_to,-event_log,-extra-cc,%multiline,\
+%attach:: -comment,-sub_components,-reason,-reported_to,-event_log,-extra-cc,%multiline,\
```

Comment:
So this will attach the subcomponents that you get from Bugzilla to the report? It’s not really useful, is it? Moreover, it’s only valid for the Red Hat instance, where the extension exists.